### PR TITLE
Repackage documentation and apply poetry updates 

### PR DESCRIPTION
**Pull request by tylerhutcherson**
Improves VL documentation with new Redis logos, poetry scripting, poetry docs dependencies, and also adds new reranking user guide.

### CODE REVIEW

File: docs/_static/js/sidebar.js
Comment (bsbodden) [inline docs/_static/js/sidebar.js:1]: Any reason the numbers on the html pages are not in order - top to bottom?

File: docs/_static/site.webmanifest
Comment (bsbodden) [inline docs/_static/site.webmanifest:1]: new brand icons I assume?

**Review by bsbodden — COMMENTED** [review]
LGTM just left a small question on the html files numbering

File: docs/_static/site.webmanifest
Comment (tylerhutcherson) [reply docs/_static/site.webmanifest:1]: Yep

File: docs/_static/js/sidebar.js
Comment (tylerhutcherson) [reply docs/_static/js/sidebar.js:1]: no reason besides not wanting to commit thousands of lines to rename haha. They were initially ordered a certain way and the custom side bar was added to allow us to dynamically change the sidebar without re committing the files themselves essentially -- probably don't need numbers on the files tbh

**Review by rbs333 — APPROVED** [review]
LGTM

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -73,6 +72,19 @@ Tests w/out rerankers:
 SKIP_RERANKERS=true poetry run test-cov
 ```
 
+### Documentation
+Docs are served from the `dcos/` directory.
```
Comment (tylerhutcherson) [inline CONTRIBUTING.md]:
```suggestion
Docs are served from the `docs/` directory.
```

**Pull request merged**